### PR TITLE
CRIMAP-372 Add metabase manifests

### PR DESCRIPTION
**Pull request by zheileman**
## Description of change
As Metabase, at least initially, is going to consume the Datastore database (RDS), it makes sense to have the deployment manifests together in this repo.

Metabase manifests will not be applied automatically as part of any datastore deploy. Again, this is just an interim place to keep them versioned. The apply for now can be done manually if something needs to change.

I considered creating a separate repo just for this, but I don't think we have yet a clear vision of how to organise this. Potentially, if we end up having an intermediate "analytics proxy service", this might sit on top of that, and we could move the manifests there.

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-372

## Notes for reviewer / how to test
Manifests have been already applied and Metabase is up and running.

### CODE REVIEW

**Review by timpeat — APPROVED** [review]
👍

File: config/kubernetes/metabase/config_map.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+kind: ConfigMap
+metadata:
+  name: configmap
+  namespace: laa-criminal-applications-metabase
+data:
+  # https://www.metabase.com/docs/latest/configuring-metabase/environment-variables
+  MAX_SESSION_AGE: "1440" # 24 hours
+  MB_ANON_TRACKING_ENABLED: "false"
+  MB_CHECK_FOR_UPDATES: "false"
+  MB_PASSWORD_LENGTH: "12"
```
Comment (willmcb) [inline config/kubernetes/metabase/config_map.yml:11]: Is this at least 12 or max of 12?

**Review by willmcb — COMMENTED** [review]
👍

File: config/kubernetes/metabase/config_map.yml (hunk 1 above)
Comment (zheileman) [reply config/kubernetes/metabase/config_map.yml:11]: At least 12.

**Pull request merged**